src/cascadia/TerminalApp/pch.h

```diff
@@ -48,3 +48,7 @@
 TRACELOGGING_DECLARE_PROVIDER(g_hTerminalWin32Provider);
 #include <telemetry\ProjectTelemetry.h>
 #include <TraceLoggingActivity.h>
+
+// JsonCpp
+#include <json-forwards.h>

```

Comment:
don't need `-forwards` if you're including the full file; saves compilation time

the _headers_ could include `forwards` and the cpp files could include the full header, if you wanted to split it up like that

Reply:
Is there a material benefit to not just dropping it in our pch? It's not like it's the significant factor in our pch size ;___;

Reply:
everything that would end up in the AST has to be parsed, and these are all overridden by the thing immediately after it :)